WebRoot/js/zimbraAdmin/common/ZaItem.js

```diff
@@ -1098,6 +1098,8 @@ ZaItem.adminHasAnyRight = function (rights) {
 
 ZaItem.formatServerTime = function(serverStr) {
 	if(serverStr) {
+		var regex = /(\d{14})(\.\d{3})Z/;
```

Comment:
you can remove second capturing group if you are not using it

Reply:
I removed it.